### PR TITLE
Fix error if inspected object have no URL #1

### PR DESCRIPTION
**Pull request by maisim**
It do the trick, let me know if I have to change other stuff related to your workflow, do it by another way...

Simon

### CODE REVIEW

File: odooshow/odooshow.py [hunk 1]
```diff
@@ -107,8 +107,12 @@ def _record_url(self, record):
         """Return a formatted link for relational records. Only supported terminals
 
         :param record record: Odoo record
-        :return str: Formatted url
+        :return str: Formatted url or None
         """
+
+        if not hasattr(record, "get_base_url"):
+            return None
```
Comment (chienandalu) [inline odooshow/odooshow.py]: I'd fallback to `self.env['ir.config_parameter'].sudo().get_param('web.base.url')` before returning `None`

Comment (maisim) [reply odooshow/odooshow.py]: What a sausage, I really did it too fast, You're right, I will update my PR

**Review by chienandalu — APPROVED** [review]
Great! Thanks a lot!

**Comment by chienandalu** [comment]
I'll be trying to prepare a new release as soon as possible with the fix

**Pull request merged**